server/mvcc/backend/testing/betesting.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package betesting
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.etcd.io/etcd/server/v3/mvcc/backend"
+	"go.uber.org/zap/zaptest"
+)
+
+func NewTmpBackendCfg(t testing.TB, bcfg backend.BackendConfig) (backend.Backend, string) {
```

Comment:
I know it is getting long, but `NewTmpBackendCfg` suggests "give me new backend config", not "give me new backend from config" which is intent if I understand it correctly.